python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -653,7 +666,12 @@ async def expand_overrides(self, overrides=None):
         self.log.debug(f"Complete overrides: {complete_overrides}")
         return complete_overrides
 
-    async def set_state(self, state, overrides=None, components=None):
+    async def set_state(
+        self,
+        state: salobj.State,
+        overrides: typing.Optional[typing.Dict[str, str]] = None,
+        components: typing.Optional[typing.List[str]] = None,
+    ) -> None:
```

Comment:
This comment refers to line 738. The signature of `traceback.format_exception` has changed in Python 3.10 leading to a failure when running MyPy. The CI build still is done with Python 3.8 and the conda build with Python 3.10 so it will be challenging to solve this.